bluesky/tests/test_new_examples.py

```diff
@@ -215,6 +220,15 @@ def read(self) -> Dict[str, Reading]:
         def locate(self) -> Location:
             return dict(setpoint=1.0, readback=1.1)
 
+        def set(self, value) -> Status:
+            self.position = value + 0.1
+            # We don't actually need a motor, but use one just to give us a Status
+            return hw.motor.set(value)
+
+        def name(self) -> str:
+            # Abstractmethod name should be defined in the subclass.
+            return ""
```

Comment:
Please can you check that the following change is sufficient to make mypy happy...
```suggestion
        name = ""
```

Reply:
With that change mypy is "happy" (has the same single error in that file for `ophyd.sim` that it did before).